cypress/page-objects/siteNavigation.js

```diff
@@ -0,0 +1,9 @@
+import BasePage from './basePage';
+
+class SiteNavigation extends BasePage {
+  getLink(linkUrl) {
+    return cy.get(`a[href="${linkUrl}"]`);
+  }
+}
```

Comment:
why just don't put it into `BasePage`?

Reply:
don't look into future
as soon as you are testing footer links and they are on every single page, then your method is the method of base page